solidity-v1/dashboard/src/components/threshold/UpgradeTokensTile.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from "react"
+import TokenAmount from "../TokenAmount"
+import Button from "../Button"
+import OnlyIf from "../OnlyIf"
+import ResourceTooltip from "../ResourceTooltip"
+import NavLink from "../NavLink"
+
+const UpgradeTokensTile = ({
```

Comment:
I wonder if we really need this component. I think It will be clearer if we render it in the parent component. Otherwise we should refactor it a little bit- eg this component only render `ResourceTooltip` but we what if we would like to display a simple tooltip? So I think all necessary components already exist and we should build this tile in the parent component. eg:

```
<section className="bg-grey-10">
<header>Wallet</header>
<List>
  <List.Content>
   <List.Item>
     Liquid KEEP <amount here>
   </List.Item>
  </List.Content>
</List>
<Button or a tag or NavLink className="<class that push the button
 to the right or wrap button
 in div and add class `text-right`>"
 >
 upgrade to t
</Button or a tag or NavLink>
</section>
```

Reply:
I disagree, I would like to keep it in the component separately since we are reusing it many times. Might be helpful when we will need to change structure of the tile - we would do that in one place in such case instead of changing it in multiple places.

I aggre with rendering the tooltips though. Fixed it in eb96bcf40236bd7e29feabdefe7001201007c4e2